Makefile

```diff
@@ -0,0 +1,163 @@
+migration-check:
+	@wget -q -O postgres-data-current.tar.gz --no-check-certificate 'https://docs.google.com/uc?export=download&id=1UwtIp7gF5uO5PfhTbOAMPpDLJ4H55m7a'
+	@tar xzf postgres-data-current.tar.gz
+	@docker-compose up -d postgres
+	@sleep 10
+	@docker-compose logs postgres | grep "database system is ready to accept connections"
+	@docker-compose down -v --remove-orphans && sudo rm -rf postgres-data-current*
+
+.PHONY: verify-configuration
+verify-configuration: # verify that configuration variables are changed correctly
+verify-configuration:
+	@python other/verify_script.py
+
+.PHONE: setup-dev
+setup-dev: # pull all images and tag them for local development
+setup-dev:
+	@for service in $(SERVICES) ; do \
+		@docker-compose pull $$service; \
+		@docker tag $(CONTAINER_REPO)/artemis-$$service:$(BUILD_TAG) artemis_$$service:latest; \
+	done
+	@if [ ! -d "local_configs" ]; then \
+		mkdir -p local_configs && \
+		mkdir -p local_configs/backend && \
+		mkdir -p local_configs/monitor && \
+		mkdir -p local_configs/frontend && \
+		cp -rn backend-services/configs/* local_configs/backend && \
+		cp -rn monitor-services/configs/* local_configs/monitor && \
+		cp -rn frontend/webapp/configs/* local_configs/frontend; \
+	fi
+
+.PHONE: setup-routinator
+setup-routinator: # create needed configuration for routinator
+setup-routinator:
+	@mkdir -p local_configs/routinator/tals
+	@sudo chown -R 1012:1012 local_configs/routinator/tals
+	@mkdir -p local_configs/routinator/rpki-repo
+	@sudo chown -R 1012:1012 local_configs/routinator/rpki-repo
+	@cp other/routinator/routinator.conf local_configs/routinator/routinator.conf
+	@sudo chown -R 1012:1012 local_configs/routinator/routinator.conf
+
+.PHONY: unittest
+unittest: # run all unit tests
+unittest:
+	@for service in detection configuration ; do \
+        PYTHONPATH=./backend-services/$$service/core pytest --cov=$$service --cov-append --cov-config=./testing/.coveragerc backend-services/$$service; \
+    done
+
+.PHONY: build
+build: # build all
+build: $(SERVICES)
+
+.PHONY: start
+start: # start local setup
+start:
+	@if [ ! -d "local_configs" ]; then \
+		mkdir -p local_configs && \
+		mkdir -p local_configs/backend && \
+		mkdir -p local_configs/monitor && \
+		mkdir -p local_configs/frontend && \
+		cp -rn backend-services/configs/* local_configs/backend && \
+		cp -rn monitor-services/configs/* local_configs/monitor && \
+		cp -rn frontend/webapp/configs/* local_configs/frontend; \
+	fi
+	@docker-compose up -d
+
+.PHONY: stop
+stop: # stop containers
+stop:
+	@docker-compose down -v --remove-orphans
+
+.PHONY: clean-db
+clean-db: # stop containers and clean volumes
+clean-db: stop
+	@sudo rm -rf postgres-data-* frontend/db/artemis_webapp.db
+
+.PHONY: release
+release: # pull and tag images for a new release
+release:
+ifneq ($(BUILD_TAG), "latest")
+	@for service in $(SERVICES) ; do \
+		@docker pull $(CONTAINER_REPO)/artemis-$$service:latest; \
```

Comment:
here too for frontend (and following lines)

Reply:
we should not use this for releasing tempfrontend

Reply:
ok, I see that this applies only to release. When we release we will push the new frontend container (and get rid of the temporary one) so it is fine. As long as that for PRs and for the current master we always push to tempfrontend when we develop stuff (until 2.0.0 within this month).